src/chunk.c

```diff
@@ -4031,6 +4039,33 @@ ts_chunk_do_drop_chunks(Hypertable *ht, int64 older_than, int64 newer_than, int3
 			data_nodes = list_append_unique_oid(data_nodes, cdn->foreign_server_oid);
 		}
 	}
+	// if we have tiered chunks cascade drop to tiering layer as well
+#if PG14_GE
+
+	if (osm_chunk_id != INVALID_CHUNK_ID)
+	{
+		hypertable_drop_chunks_hook_type osm_drop_chunks_hook =
+			ts_get_osm_hypertable_drop_chunks_hook();
+		if (osm_drop_chunks_hook)
+		{
+			ListCell *lc;
+			Dimension *dim = &ht->space->dimensions[0];
+			/* convert to PG timestamp from timescaledb internal format */
+			int64 range_start = ts_internal_to_time_int64(newer_than, dim->fd.column_type);
+			int64 range_end = ts_internal_to_time_int64(older_than, dim->fd.column_type);
+			Chunk *osm_chunk = ts_chunk_get_by_id(osm_chunk_id, true);
+			List *osm_dropped_names = osm_drop_chunks_hook(osm_chunk->table_id,
```

Comment:
Since those are osm-internal objects it makes no sense to return them to timescaledb.

Reply:
the drop_chunks api prints the names of the dropped chunks today. 
I think we need to make the call on the OSM side whether these values should be printed or not when drop_chunks is invoked.
So far, we have been consistent with the behavior across APIs.
I don't want to change the hook definition in case we decide that drop_chunks should also behave consistently.